pkg/scalers/redis_streams_scaler_test.go

```diff
@@ -22,7 +22,7 @@ func TestParseRedisStreamsMetadata(t *testing.T) {
 	testCases := []testCase{
 		{
 			name:     "with address",
-			metadata: map[string]string{"stream": "my-stream", "consumerGroup": "my-stream-consumer-group", "pendingEntriesCount": "5", "addressFromEnv": "REDIS_SERVICE", "usernameFromEnv": "REDIS_USERNAME", "passwordFromEnv": "REDIS_PASSWORD", "databaseIndex": "0", "enableTLS": "true"},
+			metadata: map[string]string{"stream": "my-stream", "consumerGroup": "my-stream-consumer-group", "pendingEntriesCount": "5", "lagCount": "5", "addressFromEnv": "REDIS_SERVICE", "usernameFromEnv": "REDIS_USERNAME", "passwordFromEnv": "REDIS_PASSWORD", "databaseIndex": "0", "enableTLS": "true"},
```

Comment:
@mikelam-us the unit tests now do not test for pendingEntriesCount configuration. We need to retain tests for both pendingEntriesCount and lagCount configurations. I would suggest you make 2 different set of unit tests for each set of configurations to test.
For example, the stream length configuration is tested here in the same file: https://github.com/mikelam-us/keda/pull/1/files#diff-f86272043db0d747a075f9434b44a09f87821e0a3b6ba3f31da92e461716b7dcR1086